repos/system_upgrade/el7toel8/actors/checktargetrepos/tests/test_checktargetrepos.py

```diff
@@ -72,8 +60,8 @@ def test_checktargetrepos_no_rhsm(monkeypatch, enable_repos, custom_target_repos
     mocked_consume = MockedConsume(_TARGET_REPOS_CUSTOM if custom_target_repos else _TARGET_REPOS_NO_CUSTOM)
     if custom_target_repofile:
         mocked_consume._msgs.append(_CUSTOM_TARGET_REPOFILE)
-    envvars = {'LEAPP_ENABLE_REPOS': 'hill,spencer'} if enable_repos else {}
-    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(envvars))
+    envars = {'LEAPP_ENABLE_REPOS': 'hill,spencer'} if enable_repos else {}
+    monkeypatch.setattr(api, 'current_actor', CurrentActorMocked(envars=envars))
```

Comment:
I can't spot the difference.

Reply:
envars don't exists anymore, but directly binded to envars kwarg inside CurrentActorMocked

Reply:
Closing it and leaving up to you, whether to change or not

Reply:
to be honest, previous 2line code was more readable for me. but that's just matter of taste. both are ok.

Reply:
Sorry, about my black formatter:) The point here was in removing envars variable and directly pass its value to the CurrentactorMocked envpars kwarg. Multiline was done just because 79 line length limitation.